src/wwwroot/template-editor/adapterService.js

```diff
@@ -183,8 +183,20 @@
           'Authorization': 'Bearer ' + apiToken
         }
       })
-      .then(function(response){
-        return { data : { images : response.data.items, count : response.data.itemsCount }};
+      .then(function(response) {
+        var items = response.data.items.map(function(item) {
+          return {
+            id : item.id,
+            image : item.image,
+            last_upload : item.last_upload,
+            name : item.name,
+            size : item.size,
+            thumbnailUrl : getHrefFromRel(item, "small"),
+            thumbnailUrl150 : getHrefFromRel(item, "large"),
```

Comment:
What do you think about using the full relation name? (in that way it will be reusable for any rel value)